tests/foreman/installer/test_installer.py

```diff
@@ -1292,6 +36,40 @@
     'tomcat',
 ]
 
+UPSTREAM_SPECIFIC_MODULES = {
+    'foreman::cli::discovery',
+    'foreman::cli::openscap',
+    'foreman::cli::rh_cloud',
+    'foreman::cli::ssh',
+    'foreman::cli::tasks',
+    'foreman::cli::templates',
```

Comment:
Ok I can add an assertion to make sure there's no cli modules in the Satellite answers

Reply:
> I think that means we shouldn't present any CLI option to install these and we should assert no `foreman::cli::*` is present.

To be clear, this also requires some engineering work to make that happen. Otherwise we're just building a test that will be guaranteed to fail.

Reply:
Oh I see you mean we need an upstream change first?

Reply:
Technically it's downstream: we need to change satellite-installer's answer files to drop them (including a migration for existing answer files). Then also reflect this in the documentation by removing all `--foreman-cli-plugin-*` mentions for satellite builds.